cmd/imagescan/imagescan.go

```diff
@@ -16,10 +16,19 @@ package imagescan
 
 import (
 	"github.com/soluble-ai/soluble-cli/pkg/tools"
+	"github.com/soluble-ai/soluble-cli/pkg/tools/hadolint"
 	"github.com/soluble-ai/soluble-cli/pkg/tools/trivy"
 	"github.com/spf13/cobra"
 )
 
 func Command() *cobra.Command {
-	return tools.CreateCommand(&trivy.Tool{})
+	c := tools.CreateCommand(&trivy.Tool{})
+	c.Use = "image-scan"
+	c.Short = "Scan a container image"
+	c.Long = `Scan a container image for vulnerabilities of OS packages or lint a container image for best practices`
```

Comment:
So hadolint doesn't scan a container image, it scans a Dockerfile.  I think this might belong under `code-scan` instead.  But if you want it here this documentation needs to be updated to say "container image or Dockerfile".

Also this is a breaking change, since `soluble image-scan --image foo` used to work but now won't.

Reply:
moved it to `code-scan`